programs/clearing_house/src/state/oracle.rs

```diff
@@ -9,6 +9,34 @@ use std::cmp::max;
 use switchboard_v2::decimal::SwitchboardDecimal;
 use switchboard_v2::AggregatorAccountData;
 
+#[derive(Default, AnchorSerialize, AnchorDeserialize, Clone, Copy, Eq, PartialEq, Debug)]
+pub struct HistOracleData {
+    // use u64?
+    pub last_oracle_price: i128,
+    pub last_oracle_conf: u128,
+    pub last_oracle_delay: i64,
+    pub last_oracle_price_twap: i128,
+    pub last_oracle_price_twap_5min: i128,
+    pub last_oracle_price_twap_ts: i64,
+
+    pub last_index_bid_price: u128,
+    pub last_index_ask_price: u128,
+    pub last_index_price_twap: u128,
+    pub last_index_price_twap_5min: u128,
+    pub last_index_price_twap_ts: i64,
+}
+
+impl HistOracleData {
```

Comment:
if this is just for testing i'd add the `#[cfg(test)]` annotation so it only is included in test build